package.json

```diff
@@ -89,7 +86,6 @@
 		"typeorm": "0.2.28",
 		"typeorm-naming-strategies": "^2.0.0",
 		"utf-8-validate": "^6.0.3",
-		"ws": "^8.13.0",
```

Comment:
Why `ws` gone? DJS still uses it

Reply:
Remnant from when we had a `ws` handler for the music dashboard. It's a direct dependency of discord.js anyways.